src/config.js

```diff
@@ -81,6 +102,7 @@ export function buildConfig (options: ConfigOptions = {}): Config {
     root: path.join(__dirname, '..'),
     platform: os.platform(),
     arch: os.arch(),
+    _skipAnalytics: false,
```

Comment:
@KarateCowboy continuing comment from above: could this be done like `debug` where it sets the default from the `skipAnalytics()` function?